### DISCUSSION TITLE
way to calculate current draw with undervolt and higher clocks?

### DISCUSSION THREAD

**Discussion by WobbleTheHutt** [hanai3Bi/Switch-OC-Suite#24] [General]
my SOC happily works with undervolt level 2 and the system doesn't turn off from overcurrent protection when using 1228mhz on the GPU and I was wondering how I can calculate what the current draw would be. I have seen snippets of table based on speed0 but I can't seem to find the actual document.

**Comment by hanai3Bi** [comment]
it's not easy to get accurate draw, but you can use Status Moniter Overlay, or external power meter to see power draw

**Comment by WobbleTheHutt** [comment]
Issue is current loader.kip locks gpu clocks to 998 even if unsafe is set.
I was only seeing bursts of about 3amp draw with that and 1963 cpu though
while forcing totk into docked mode with reverse nx.


Anyway to bypass the sanity checks while handheld so I can briefly check
power draw?
